src/Tasks/Microsoft.Common.CurrentVersion.targets

```diff
@@ -350,6 +352,13 @@ Copyright (C) Microsoft Corporation. All rights reserved.
     <FinalDocFile Include="@(DocFileItem->'$(OutDir)%(Filename)%(Extension)')"/>
   </ItemGroup>
 
+  <ItemGroup Condition="'$(Deterministic)' == 'true'">
```

Comment:
Needs an explicit "no ref asm" off switch, as well as defaulting for `Deterministic`.